Sources/Overload/OvEditor/include/OvEditor/Settings/EditorSettings.h

```diff
@@ -75,9 +75,9 @@ namespace OvEditor::Settings
 
 		inline static Property<bool> ShowGeometryBounds = { false };
 		inline static Property<bool> ShowLightBounds = { false };
-		inline static Property<bool> EditorFrustumGeometryCulling = { false };
-		inline static Property<bool> EditorFrustumLightCulling = { false };
-		inline static Property<bool> DebugFrustumCulling = { false };
+		inline static Property<bool> EditorFrustumGeometryCulling = { true };
+		inline static Property<bool> EditorFrustumLightCulling = { true };
+		inline static Property<bool> DebugFrustumCulling = { true };
```

Comment:
This one should actually stay set to `false`. `DebugFrustumCulling` can be toggled to apply the frustum culling of the main scene camera to the scene view, and is only useful to preview how the main camera will apply frustum culling in game